### DISCUSSION TITLE
Not able to publish project in staging for Observation and survey

### DISCUSSION THREAD

**Discussion by ramkumar-ns-dev** [Sunbird-coKreat/Community#81] [Bugs]
**Dock-staging**
We have created some projects in dock-staging by using **Observation and survey** category to test the complete creation flow, And we completed the following steps successfully

1. Project creation (sourcing admin)
2. Nomination (contributor admin)
3. Approve-nomination (sourcing admin)
4. contributor submitting for review (contributor)
5. contributor approving review (contributor reviewer)

once the contributor reviewer approves the contribution, the request is coming to source admin to approve, But following are the issues we are facing

1. Status is showing as **"Review pending"** instead of **"Approval pending"**.
2. We are not able to see the content and also not able publish the content.

Can anybody help on this. @arun1205

cc: @kiranharidas187

**Comment by Aparna-karri591** [comment]
@ramkumar-pacewisdom it looks like pipeline is blocked, @gandham-santhosh @santhosh-tg can you please check this
cc: @vinukumar-vs @manimegalai-smohan

**Reply by ramkumar-ns-dev** [reply]
@Aparna-karri591 Thank you. @gandham-santhosh @santhosh-tg kindly please check and let me know if anything needed from my side.

**Reply by ramkumar-ns-dev** [reply]
@Aparna-karri591 would you please name the pipeline which is blocked ?

**Reply by Aparna-karri591** [reply]
@tsprasath @ramkumar-pacewisdom can you Please check Question set pipeline

**Reply by Aparna-karri591** [reply]
@vaivk369 can u pls look into this

**Reply by ramkumar-ns-dev** [reply]
@Aparna-karri591 issue resolved by restarting Question set pipeline, Thanks a lot.
